inference-engine/ie_bridges/java/tests/README.md

```diff
@@ -0,0 +1,26 @@
+## Build
+Create an environment variable with Inference Engine installation path:
+```bash
+export IE_PATH=/path/to/openvino/bin/intel64/Release/lib/
+```
+
+To create java wrappers tests add `-DENABLE_JAVA=ON` and `-DENABLE_TESTS=ON` flags in cmake command while building openvino:
+```bash
+cd openvino/build
+cmake -DCMAKE_BUILD_TYPE=Release -DENABLE_JAVA=ON -DENABLE_TESTS=ON ..
+make
+```
+
+Link to repo with [test data](https://github.com/openvinotoolkit/testdata.git)
+
+## Running
+Add library path for model java before running:
+```bash
+export MODELS_PATH=/path/to/testdata
+```
+
+To run tests use:
+```bash
+cd ..
+java -cp ".:${IE_PATH}/*" OpenVinoTestRunner
```

Comment:
Where it's located? Shall we change directory?

Reply:
Now the ```OpenVinoTestRunner.class``` is inside of ```openvino_tests.jar```. We can find the jar using the```${IE_PATH}```, so 
we can run this command from any directory.